### ISSUE TITLE
Drag And Drop Features

### ISSUE THREAD

**Issue by PrashantUnity** [martinstoeckli/SilentNotes#257]
Hi @martinstoeckli Thankyou for this Application, Yesterday I Discovered this on FDroid and love this so much one its easy to use and best part is  its open source.

One this i miss is Dragging features. Could You please implement Freature of Dragging for rearranging.

Given that the project uses MudBlazor, which offers built-in dragging features as demonstrated in their [Dropzone component](https://mudblazor.com/components/dropzone#miscellaneous-chess-board), implementing this feature could be a valuable addition to the application.

**Comment by martinstoeckli** [comment]
Hello @PrashantUnity thanks a lot for your suggestion.

Actually I just replaced the Drag&Drop feature with the new ordering toolbar. Before version 8.0, I used a JavaScript library to do the sorting, but on mobile devices it did not work reliable (it was laggy and sometimes hanged). The component you described looks interesting, I would have to do some testing though. And another step I surely want to do first, is updating to MudBlazor 4 (I actually already started, but then time was running out).

**Comment by martinstoeckli** [comment]
Maybe the problem could be also solved, by adding additional buttons to move a whole row of notes, when the application is shown on a wide screen? What do you think?

**Comment by PrashantUnity** [comment]
were you using same Library https://github.com/SortableJS/Sortable
i found while i was watching Burke Learns Blazor - Drag and Drop and DOM events!
https://www.youtube.com/live/OLCnfudwNSc?si=gD0jTqYxGBJLLpdb

from you source code i am guessiong we need to do some changes in **NoteRepositoryPage.razor** line 156

Testing can be pain 😭😭😭

**Comment by martinstoeckli** [comment]
Yes this is the place where the visible notes are enumerated 👍and yes testing is really difficult due to the sheer amout of possible environments. Usually I test with Android 7, 10, 13 and 14 before a release is published, and it is more common than not, that there are some hickups on one of those devices. 😥
